CHANGELOG.md

```diff
@@ -2,14 +2,19 @@
 
 ## [v0.4.0](https://github.com/vmware/terraform-provider-vcf/releases/tag/v0.4.0)
 
-> Release Date: Aug X 2023
+> Release Date: Aug 26th 2023
 
 BREAKING CHANGES:
 
 * Removed the attribute "host_id" from the "vcf_host" resource and replaced it with just "id" as per Terraform standard practice. This way users can refer to the Host UUID (in cluster host spec for example) in the standard way, e.g. "vcf_host.host1.id"
+* Replaced attribute "nsx_cluster_ref" from the "vcf_domain" datasource with a richer "nsx_configuration", that additionally contains IPs, Names and DNS Names of NSX-T Manager Nodes
+* Replaced attribute "vcenter_fqdn" and "vcenter_id from the "vcf_domain" datasource with "vcenter" subresource, that contains "id" and "fqdn" attributes. 
```

Comment:
for consistency, should this be "vcenter_configuration" just like "nsx_configuration" above?

Reply:
Sure! However keep in mind this refers to the domain **datasource**, we can change that in the domain **resource** as well, but that will be a breaking change. With this current naming there is parity between datasource and resource.

Reply:
Given this is already a breaking change, I think it's ok to fix things for consistency and correctness.